Transformers/ServiceNowFormDataTransformer.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace EMS\SubmissionBundle\Transformers;
+
+class ServiceNowFormDataTransformer extends AbstractFormDataTransformer
+{
+    public function transform(): array
+    {
+        $data = $this->form->getData();
+
+        return $this->mapNewFieldTimeFormat($data);
+    }
+
+    private function mapNewFieldTimeFormat(array $data): array
+    {
```

Comment:
I think this function should not exist. The mapper should loop over all fields, and change every field when appropriate. The transform function can be the loop and call the updates whenever needed (see my other comment below)